schemas/schemas/infoQuestion.json

```diff
@@ -96,6 +96,10 @@
             "description": "Options that define how the question will work, specific to the individual question type.",
             "type": "object"
         },
+        "parameters": {
+            "description": "Information on parameters that the question is prepared to accept from questionParams sources.",
+            "type": "object"
+        },
```

Comment:
I may be mistaken, but I don't see this being used anywhere, are you planning to filter the parameters for a question?

Reply:
I think I was experimenting with having questions specify what parameters they recognized, but that ended up being very tedious.